doozer/doozerlib/backend/rebaser.py

```diff
@@ -0,0 +1,1549 @@
+import copy
+import hashlib
+import io
+import logging
+import os
+import re
+import shutil
+from pathlib import Path
+from typing import Dict, List, Optional, Set, Tuple, cast
+
+import bashlex
+import bashlex.errors
+import yaml
+from artcommonlib import exectools, release_util
+from artcommonlib.model import ListModel, Missing
+from dockerfile_parse import DockerfileParser
+
+from doozerlib import constants, util
+from doozerlib.backend.build_repo import BuildRepo
+from doozerlib.brew import BuildStates
+from doozerlib.image import ImageMetadata
+from doozerlib.record_logger import RecordLogger
+from doozerlib.repos import Repos
+from doozerlib.runtime import Runtime
+from doozerlib.source_modifications import SourceModifierFactory
+from doozerlib.source_resolver import SourceResolution, SourceResolver
+
+LOGGER = logging.getLogger(__name__)
+
+
+CONTAINER_YAML_HEADER = """
+# This file is managed by Doozer: https://github.com/openshift-eng/art-tools/tree/main/doozer
+# operated by the OpenShift Automated Release Tooling team (#forum-ocp-art on CoreOS Slack).
+
+# Any manual changes will be overwritten by Doozer on the next build.
+#
+# See https://source.redhat.com/groups/public/container-build-system/container_build_system_wiki/odcs_integration_with_osbs
+# for more information on maintaining this file and the format and examples
+
+---
+"""
+
+
+# Doozer used to be part of OIT
```

Comment:
Maybe we can remove oit references and replace them with something better?

Reply:
Sounds good to me. I haven't got chance to change this after moving this part of code from ImageDistgitRepo class.